pkg/apis/litmuschaos/v1alpha1/chaosresult_types.go

```diff
@@ -43,6 +50,8 @@ type TestStatus struct {
 	Phase string `json:"phase"`
 	// Verdict defines whether an experiment result is pass or fail
 	Verdict string `json:"verdict"`
+	// FailStep defines step where the experiments fails
+	FailStep string `json:"failStep,omitempty"`
```

Comment:
Check for this omitempty also

Reply:
failStep is only meaningful if the exp failed, not in case of pass